dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# A shell script to start the Node.js servers 
+# along with the mongod process, all of which
+# are necessary for RocketMeet-client to function.
+
+# Get the init system of the developer's platform.
+initSystem=`ps --no-headers -o comm 1`
+
+# Perform a check and start mongod.
+if [ "$initSystem" = "init" ]
+then
+    sudo service mongod start
+else 
+    sudo systemctl start mongod
+fi
+
+# Start RocketMeet-server
+npm run dev --prefix ../RocketMeet-server/ &
+
+# Start RocketMeet-mailer
+npm run dev --prefix ../RocketMeet-mailer/ &
+
+# Start RocketMeet-client
+npm run dev
```

Comment:
Another thing, doesn't it enforces user to have a specific folder directory and naming? I hope that's okay

Reply:
You may read the conversation from this link.
[ https://github.com/RocketMeet/RocketMeet-client/issues/65#issuecomment-794998674 ]